OrbitCore/CallstackData.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "CallstackData.h"
+
+#include "Callstack.h"
+#include "Capture.h"
+#include "SamplingProfiler.h"
+
+using orbit_client_protos::CallstackEvent;
+
+void CallstackData::AddCallstackEvent(CallstackEvent&& callstack_event) {
```

Comment:
Can you add a comment at the declaration of this function, that it assumes that the callstack hash is known?

Reply:
Done

Reply:
I think we prefer value arguments for sinks, and move them explicitly on the call side, such that you can see on the caller, that the object becomes invalid after the call.

Reply:
Done